setup.py

```diff
@@ -348,6 +350,16 @@ def finalize_options(self):
     libs.append(providers_cuda_or_rocm)
     libs.append(providers_tensorrt_or_migraphx)
     libs.append(providers_cann)
+    # QNN
+    qnn_deps = [
+        "libQnnCpu.so",
+        "libQnnHtp.so",
+        "libQnnSaver.so",
+        "libQnnSystem.so",
+        "libHtpPrepare.so",
+        "onnxruntime_qnn_ctx_gen",
```

Comment:
only need libQnnHtp.so for onnxruntime_qnn_ctx_gen

Reply:
can keep the necessary libs to enable full functionality